packages/core/src/internal/utils/dom.ts

```diff
@@ -238,3 +238,10 @@ export function getShadowRootOrElse(hostEl: HTMLElement, fallbackEl?: HTMLElemen
   const fallTo = fallbackEl ? fallbackEl : hostEl;
   return (hostEl.shadowRoot ? hostEl.shadowRoot : fallTo) as HTMLElement;
 }
+
+export type BooleanProperty = string | boolean | null | undefined;
+
+/** Coerces attribute/property value to a boolean */
+export function coerceBooleanProperty(value: any): boolean {
+  return value !== null && value !== undefined && `${value}` !== 'false';
```

Comment:
Making sure that empty string being true is expected. That's what will happen. I believe that is expected but probably want it in the unit tests to be explicit about it.